src/Consolonia.Themes/Templates/Controls/ScrollBar.axaml

```diff
@@ -169,10 +172,18 @@
         <Style Selector="^:vertical">
             <Setter Property="Width"
                     Value="1" />
+            <Setter Property="SmallChange"
+                    Value="1" />
+            <Setter Property="LargeChange"
```

Comment:
It's not for sure this ScrollBar is used by a scroll viewer upper in the tree, It looks like this value should be set in a style in a ScrollViewer theme.

Reply:
good call, I moved this up to the scrollviewer